addons/project/wizard/project_share_wizard_template.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ProjectShareWizardTemplate(models.TransientModel):
+    _name = 'project.share.wizard.template'
+    _description = 'Project Sharing Template'
+
+    partner_id = fields.Many2one('res.partner', string='recipient')
+    project_id = fields.Many2one('project.project', string='project')
+    is_portal = fields.Boolean(default=True)
+
+    def get_share_link(self):
+        if self.is_portal:
+            return f"{self.project_id.get_base_url() + self.project_id._get_share_url(redirect=True, pid=self.partner_id.id)}"
```

Comment:
Same question here. If you need a conversion to string, I would use `str` for readability